locales/en.default.schema.json

```
@@ -262,6 +262,16 @@
       "settings": {
         "animations_reveal_on_scroll": {
           "label": "Reveal sections on scroll"
+        },
+        "animations_hover_elements": {
+          "options__1": {
+            "label": "None"
+          },
+          "options__2": {
+            "label": "3D Lift"
+          },
+          "label": "Hover Effect",
+          "info": "Applies to cards and buttons."
```

```suggestion
        "animations_hover_elements": {
          "options__1": {
            "label": "None"
          },
          "options__2": {
            "label": "3D lift"
          },
          "label": "Hover effect",
          "info": "Affects cards and buttons."
```

I am wondering for the setting if we are planning to add additional `Hover effect` very soon? I know we were considering lifting elements but just vertically at some point. I took a quick look at what it could look like. Otherwise should it just be a [checkbox until we add more](https://screenshot.click/04-34-801wg-7hw4a.png)?

I am doing content suggestions based on common patterns observed for other settings labels and info text.

---

Thanks! I planning a quick followup to this PR to add the vertical lift effect, so I think using the dropdown here makes sense. It'll be great to have that set up so we can just plug into it moving forward.